### PR TITLE
add indicator handling in config

### PR DESCRIPTION
**Pull request by DenisScherbakov**
- highlight cells with different colors as configured
- fix for related import in pivot_table for Windows
- add possibility to store normalized data as 'raw' excel sheet

### CODE REVIEW

File: bearysta/aggregate.py [hunk 1]
```diff
@@ -723,7 +723,7 @@ def format_column(self, col, boundaries, colors):
             indices += (col > b)
         indices[col.isnull()] = -1
         indices = indices.astype('int64')
-
+
```
[the remark this answers is not in the document]
Comment (DenisScherbakov) [reply bearysta/aggregate.py:726]: removed

File: bearysta/aggregate.py [hunk 2]
```diff
@@ -799,16 +799,25 @@ def create_pandas_pivot_table(self, df, excel=None):
                                         'variant {}'.format(variant))
                     continue
 
-                pt.to_excel(writer, sheet_name='summary', startrow=position)
+                if self['indicator'] is not None:
+                    for col in self['indicator']:
+                        ss = pt.style.apply(self.format_column, boundaries=col['ranges'], colors=col['colors'], subset=col['column'])
```
[the remark this answers is not in the document]
Comment (DenisScherbakov) [reply bearysta/aggregate.py:804]: to be renamed: "pt" -> "temp_pt" as pivot_table without final formatting (highlighting of cells). "ss" -> "pt" as final pivot_table to be stored in excel

File: bearysta/aggregate.py [hunk 3]
```diff
@@ -799,16 +799,25 @@ def create_pandas_pivot_table(self, df, excel=None):
                                         'variant {}'.format(variant))
                     continue
 
-                pt.to_excel(writer, sheet_name='summary', startrow=position)
+                if self['indicator'] is not None:
```
[the remark this answers is not in the document]
Comment (DenisScherbakov) [reply bearysta/aggregate.py:802]: OK

File: bearysta/aggregate.py [hunk 4]
```diff
@@ -799,16 +799,25 @@ def create_pandas_pivot_table(self, df, excel=None):
...
+                    for col in self['indicator']:
+                        ss = pt.style.apply(self.format_column, boundaries=col['ranges'], colors=col['colors'], subset=col['column'])
+                else:
+                    ss = pt
```
[the remark this answers is not in the document]
Comment (DenisScherbakov) [reply bearysta/aggregate.py:806]: OK

**Review by Vyacheslav-Smirnov — APPROVED** [review]

**Pull request merged**